test/kafka-ingest-open-loop/setup.td

```diff
@@ -0,0 +1,49 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+#
+# Setup for performing load tests for persistent upsert sources.
+#
+
+$ kafka-create-topic topic=load-test partitions=4
+
+# Mirror mz_source_info and use to keep track of how much data has been ingested
+# at all times.
+> CREATE MATERIALIZED VIEW records_ingested AS
+  SELECT SUM("offset") FROM mz_source_info WHERE
+  source_name = 'testdrive-load-test-${testdrive.seed}';
+
+# Alter the underlying index to keep track of historical data.
+> ALTER INDEX records_ingested_primary_idx SET(logical_compaction_window="off");
+
+# Sleep enough so that no one asks for times not already present in the index.
+> SELECT mz_internal.mz_sleep(2)
+<null>
+
+> CREATE SOURCE load_test
+  FROM KAFKA BROKER '${testdrive.kafka-addr}' TOPIC 'testdrive-load-test-${testdrive.seed}'
```

Comment:
should we hardcode `-1` here to match the hardcoded value in `send_records`?

Reply:
sorry, I don't follow. isn't this the only user of this script?

Reply:
yes thats true but sometimes I'll use the testdrive file within a workflow to manually do what the workflow is doing. I've definitely done that in the past with the exactly-once-sinks test, and I could see doing something like this in order to have measure mz with instruments on my laptop. When running in that kind of environment its nice to be able to change the seed so you don't have to tear down the kafka topic as you iterate on stuff

Reply:
definitely a niche usecase. My second thought here is that using the seed is idiomatic testdrive usage, and its actually a hack that we hardcode the seed in the mzcompose.py, and perhaps instead that code should invoke testdrive with `consistent_seed=True` and have a method that lets someone get the seed. I think the second thought is more convincing

Reply:
Tangentially related: I'd like to very soon move this sort of stuff out of testdrive entirely. It should be possible to write this all in the Python workflow file. E.g.:

```py
kafka = c.kafka_client("kafka")
kafka.create_topic(topic="foo", partitions=4)
c.sql("CREATE SOURCE...")
c.sleep(5)
c.sql("...")
```

Then you wouldn't need to worry about the seed at all.